app/models/website.rb

```diff
@@ -1,9 +1,16 @@
 class Website < ActiveRecord::Base
   validates_presence_of :long_url
+  validate :long_url_must_begin_with_protocol
   after_create :shorten_url
 
   def shorten_url
     update_attributes(short_url: id)
   end
 
+  def long_url_must_begin_with_protocol
+    unless long_url.start_with?('http://', 'https://')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.